gamemodes/horde/gamemode/perks/medic/medic_base.lua

```diff
@@ -14,15 +14,35 @@ PERK.Params = {
 }
 
 PERK.Hooks = {}
+
+local function getHealthColor( health )
+    local hue = math.Clamp(( health / 100 ) * 120, 0, 175)
```

Comment:
have you considered using a cache for this compared to constantly calculating it? might help with performance, not sure how much mem it'd take up though